react-native-camera for iOS doesn't work with latest react-native v0.40.0

### Steps to reproduce
1. Initialize new project using latest react-native v0.40.0
2. npm install react-native-camera
3. react-native link
4. run-ios

### Expected behaviour
Boilerplate app builds and runs

### Actual behaviour
Compilation errors:
`
In file included from [...]/node_modules/react-native-camera/ios/RCTCamera.m:4:
../../react-native/React/Base/RCTLog.h:42:3: error: redefinition of enumerator 'RCTLogLevelInfo'
`

### react-native-camera
**Version**:     "react-native-camera": "^0.4.0",

--- a/ios/RCTCamera.m
+++ b/ios/RCTCamera.m
@@ -1,9 +1,9 @@
-#import "RCTBridge.h"
+#import <React/RCTBridge.h>
 #import "RCTCamera.h"
 #import "RCTCameraManager.h"
-#import "RCTLog.h"
-#import "RCTUtils.h"
-#import "RCTEventDispatcher.h"
+#import <React/RCTLog.h>
+#import <React/RCTUtils.h>
+#import <React/RCTEventDispatcher.h>
 
 #import "UIView+React.h"
 

--- a/ios/RCTCameraManager.h
+++ b/ios/RCTCameraManager.h
@@ -1,4 +1,4 @@
-#import "RCTViewManager.h"
+#import <React/RCTViewManager.h>
 #import <AVFoundation/AVFoundation.h>
 
 @class RCTCamera;

--- a/ios/RCTCameraManager.m
+++ b/ios/RCTCameraManager.m
@@ -1,10 +1,10 @@
 #import "RCTCameraManager.h"
 #import "RCTCamera.h"
-#import "RCTBridge.h"
-#import "RCTEventDispatcher.h"
-#import "RCTUtils.h"
-#import "RCTLog.h"
-#import "UIView+React.h"
+#import <React/RCTBridge.h>
+#import <React/RCTEventDispatcher.h>
+#import <React/RCTUtils.h>
+#import <React/RCTLog.h>
+#import <React/UIView+React.h>
 #import "NSMutableDictionary+ImageMetadata.m"
 #import <AssetsLibrary/ALAssetsLibrary.h>
 #import <AVFoundation/AVFoundation.h>

